app/controllers/application_controller.rb

```diff
@@ -1583,6 +1583,7 @@ def get_view(db, options = {})
       :association           => association,
       :filter                => get_view_filter(options[:filter]),
       :sub_filter            => get_view_process_search_text(view),
+      :supports_filter       => options[:supports_filter],
```

Comment:
Okiedoke @Fryguy and @durandom the option has been renamed as suggested.  Any chance of merging?  Thanks much.